hazelcast/src/test/java/com/hazelcast/map/nearcache/NearCacheTestSupport.java

```diff
@@ -117,12 +119,15 @@ public void run() {
     protected NearCacheConfig newNearCacheConfigWithEntryCountEviction(EvictionPolicy evictionPolicy, int size) {
         return newNearCacheConfig()
                 .setCacheLocalEntries(true)
-                .setMaxSize(size)
-                .setEvictionPolicy(evictionPolicy.name());
+                .setEvictionConfig(new EvictionConfig(size, ENTRY_COUNT, evictionPolicy));
     }
 
     protected NearCacheConfig newNearCacheConfig() {
-        return new NearCacheConfig();
+        NearCacheConfig nearCacheConfig = new NearCacheConfig();
+
+        nearCacheConfig.getEvictionConfig().setSize(NearCacheConfig.DEFAULT_MAX_SIZE);
```

Comment:
Had to set the default value from the deprecated `max-size` configuration here, since the new `EvictionConfig` has a way smaller default (`10000` vs. `Integer.MAX_VALUE`). This is a behavior change users might run into!